docs/content/doc/usage/issue-references.en-us.md

```diff
@@ -0,0 +1,166 @@
+of creating the reference.
+* The reference is inside a commit message.
+* The reference is posted as part of the pull request description.
+
+In the last case, the issue will be closed or reopened only if the merger
+of the pull request has permissions to do so.
+
+Additionally, only pull requests and commit messages can create an action,
+and only issues can be closed or reopened this way.
+
+The default _keywords_ are:
+
+* **Closing**: close, closes, closed, fix, fixes, fixed, resolve, resolves, resolved
+* **Reopening**: reopen, reopens, reopened
+
+## External Trackers
+
+Gitea supports the use of external issue trackers, and references to issues
+hosted externally can be created in pull requests. However, if the external
+tracker uses numbers to identify issues, they will be indistinguishable from
+the pull requests hosted in Gitea. To address this, Gitea allows the use of
+the `!` marker to identify pull requests. For example:
+
+> This is issue [#1234](#), and links to the external tracker.
+
+> This is pull request [!1234](#), and links to a pull request in Gitea.
+
+The `!` and `#` can be used interchangeably for issues and pull request _except_
+for this case, where a distinction is required.
+
+## Issues and Pull Requests References Summary
+
+| Reference in User1/Repo1  | Repo1 issues are external | RepoZ issues are external | Should render                    |
+|---------------------------|:-------------------------:|:-------------------------:|----------------------------------|
+| `#1234`                   |     no                    |    N/A                    | A link to issue/pull 1234 in `User1/Repo1` |
+| `!1234`                   |     no                    |    N/A                    | A link to issue/pull 1234 in `User1/Repo1` |
+| `#1234`                   |    yes                    |    N/A                    | A link to _external issue_ 1234 for `User1/Repo1` |
+| `!1234`                   |    yes                    |    N/A                    | A link to _PR_ 1234 for `User1/Repo1` |
+| `User1/Repo1#1234`        |     no                    |    N/A                    | A link to issue/pull 1234 in `User1/Repo1` |
+| `User1/Repo1!1234`        |     no                    |    N/A                    | A link to issue/pull 1234 in `User1/Repo1` |
+| `User1/Repo1#1234`        |    yes                    |    N/A                    | A link to _external issue_ 1234 for `User1/Repo1` |
+| `User1/Repo1!1234`        |    yes                    |    N/A                    | A link to _PR_ 1234 for `User1/Repo1` |
+| `UserZ/RepoZ#1234`        |    N/A                    |     no                    | A link to issue/pull 1234 in `UserZ/RepoZ` |
+| `UserZ/RepoZ!1234`        |    N/A                    |     no                    | A link to issue/pull 1234 in `UserZ/RepoZ` |
+| N/A                       |    N/A                    |    yes                    | A link to _external issue_ 1234 for `UserZ/RepoZ` |
```

Comment:
Maybe _Not supported_ is more clear than _N/A_ here.

Reply:
@davidsvantesson done. I've also renamed the file for the page as it's no longer specific for issues.

[Link to rendered version](https://github.com/guillep2k/gitea/blob/fix-9088/docs/content/doc/usage/linked-references.en-us.md) for easier checking.